### DISCUSSION TITLE
Prevent a PVC from being resized

### DISCUSSION THREAD

**Discussion by umeshbitgo** [topolvm/pvc-autoresizer#282] [Q&A] [answered]
Hi,
I want to ignore a few PVCs in my cluster from being resized by pvc-autoresizer. is there an annotation to to ignore PVCs? I initially thought that if we don't annotate the PVC then it would be ignored but as per the documentation:
The PVC can optionally have resize.topolvm.io/threshold, resize.topolvm.io/inodes-threshold and resize.topolvm.io/increase annotations. (If they are not given, the default value is 10%.) It seems even if I don't annotate the PVC, this tool would set a default value of 10% for each of these annotations.

**Comment by daichimukai** [comment]
Hi @umeshbitgo,

Do your PVCs which you want pvc-autoresizer ignores have the `resize.topolvm.io/storage_limit` annotation? pvc-autoresizer just ignores a PVC unless the PVC has that annotation.

**Reply by umeshbitgo** [reply]
Yes, I have this annotation on the PVC and is set to a number. I don't want it to resize further, I was thinking of setting it to zero, its as good as not having this annotation at all on the PVC.
It would be better if we had this annotation `resize.topolvm.io/enabled ` at the PVC level.

**Reply by daichimukai** [reply, marked as answer]
I'm not sure yet why you need a new annotation. It seems to me that just removing `resize.topolvm.io/storage_limit` annotation is the right and simple way.
